### ISSUE TITLE
Undocumented and untested functions in Devel::QuickCover

### ISSUE THREAD

**Issue by jkeenan** [p5pclub/devel-quickcover#14]
Devel::QuickCover (i.e., lib/Devel/QuickCover.pm) source code contains two functions, set_output_directory() and set_metadata(), neither of which are documented.

``` sh
[devel-quickcover] 34 $ ack set_output_directory .
lib/Devel/QuickCover.pm
43:sub set_output_directory {
[devel-quickcover] 35 $ ack set_metadata .
lib/Devel/QuickCover.pm
50:sub set_metadata {
```

Nor are these functions exercised in your test suite (as I learned by running the distribution through Devel::Cover).

What purpose do they serve?  If they should be used, they should be publicly documented and tested.

Thank you very much.
Jim Keenan

**Comment by gonzus** [comment]
You are right, need to document and test these two. Thanks for bringing it to my attention, will do so presently.

Cheers!
